pkg/operator/storageclasscontroller/storageclasscontroller.go

```diff
@@ -86,6 +108,7 @@ func (c *StorageClassController) Sync(ctx context.Context, connection *vclib.VSp
 			utils.InstallErrorMetric.WithLabelValues(string(syncResult.CheckStatus), clusterCondition).Set(1)
 			return syncResult, checks.ClusterCheckAllGood
 		}
+		c.policyName = policyName
 
 		err := c.syncStorageClass(ctx, policyName, scState)
```

Comment:
optional: syncStorageClass could now read from c.policyName instead of a separate argument.